tsconfig.base.json

```diff
@@ -26,6 +26,7 @@
     "paths": {
       "@fxa/payments/capability": ["libs/payments/capability/src/index.ts"],
       "@fxa/payments/cart": ["libs/payments/cart/src/index.ts"],
+      "@fxa/payments/cart/*": ["libs/payments/cart/src/lib/*"],
```

Comment:
Is this necessary? We've been trying to use barrel files for exports from `libs` as much as possible.

Reply:
nope! initially done in favor of package sizes, but removing from this PR